homeassistant/components/upnp/sensor.py

```diff
@@ -80,17 +120,26 @@ async def async_setup_entry(
 
     LOGGER.debug("Adding sensors")
 
-    sensors = [
-        RawUpnpSensor(coordinator, SENSOR_TYPES[BYTES_RECEIVED]),
-        RawUpnpSensor(coordinator, SENSOR_TYPES[BYTES_SENT]),
-        RawUpnpSensor(coordinator, SENSOR_TYPES[PACKETS_RECEIVED]),
-        RawUpnpSensor(coordinator, SENSOR_TYPES[PACKETS_SENT]),
-        DerivedUpnpSensor(coordinator, SENSOR_TYPES[BYTES_RECEIVED]),
-        DerivedUpnpSensor(coordinator, SENSOR_TYPES[BYTES_SENT]),
-        DerivedUpnpSensor(coordinator, SENSOR_TYPES[PACKETS_RECEIVED]),
-        DerivedUpnpSensor(coordinator, SENSOR_TYPES[PACKETS_SENT]),
-    ]
-    async_add_entities(sensors)
+    entities = []
+    entities.append(
+        RawUpnpSensor(
+            coordinator=coordinator,
+            entity_description=entity_description,
+        )
+        for entity_description in SENSOR_ENTITY_DESCRIPTIONS[RAW_SENSOR]
+        if coordinator.data.get(entity_description.key) or False
+    )
+
+    entities.append(
+        DerivedUpnpSensor(
+            coordinator=coordinator,
+            entity_description=entity_description,
+        )
+        for entity_description in SENSOR_ENTITY_DESCRIPTIONS[DERIVED_SENSOR]
+        if coordinator.data.get(entity_description.key) or False
```

Comment:
```suggestion
        if coordinator.data.get(entity_description.key) is not None
```

pylint does not like the or False